sdk/storage/storage-blob/package.json

```diff
@@ -141,7 +141,7 @@
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^4.0.1",
+    "@azure/identity": "~4.4.0",
```

Comment:
It Should be OK as this is dev dependency.
However, we should remove the limitation when the issue on the new version of "@azure/identity" is resolved.

Reply:
logged https://github.com/Azure/azure-sdk-for-js/issues/30488